includes/attendee-repository.php

```diff
@@ -37,25 +43,45 @@ public function remove_attendee( int $event_id, int $user_id ): void {
 			throw new Exception( 'invalid user id' );
 		}
 
-		$event_ids = get_user_meta( $user_id, self::USER_META_KEY, true );
-		if ( ! $event_ids ) {
-			$event_ids = array();
-		}
-
-		if ( isset( $event_ids[ $event_id ] ) ) {
-			unset( $event_ids[ $event_id ] );
-		}
-
-		update_user_meta( $user_id, self::USER_META_KEY, $event_ids );
+		global $wpdb, $gp_table_prefix;
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		$wpdb->query(
+			$wpdb->prepare(
```

Comment:
Could we use `$wpdb->delete()`? Then we don't need all the above disables.

Reply:
We're now using `$wpdb->delete()`. It allows for removing `WordPress.DB.PreparedSQL.InterpolatedNotPrepared` but not the other ones.